shared/public/Value.h

```diff
@@ -111,18 +111,21 @@ class FeatureContext {
 public:
     uint64_t identifier;
     vtzero::GeomType geomType;
+    bool hasCustomId = false;
 
     FeatureContext() {}
 
     FeatureContext(const FeatureContext &other)
         : propertiesMap(other.propertiesMap)
         , geomType(other.geomType)
-        , identifier(other.identifier) {}
+        , identifier(other.identifier)
+        , hasCustomId(other.hasCustomId) {}
 
     FeatureContext(FeatureContext &&other)
         : propertiesMap(std::move(other.propertiesMap))
         , geomType(other.geomType)
-        , identifier(other.identifier) {}
+        , identifier(other.identifier)
+        , hasCustomId(other.hasCustomId) {}
 
     FeatureContext(vtzero::GeomType geomType,
                    mapType propertiesMap,
```

Comment:
This constructor takes an identifier as parameter, but we don't know if it is a unique "custom" ID or not. For our purposes, we should conservatively assume that it is unique.